mmv1/products/compute/NetworkEndpoints.yaml

```
@@ -117,7 +117,7 @@ properties:
             The name for a specific VM instance that the IP address belongs to.
             This is required for network endpoints of type GCE_VM_IP_PORT.
             The instance must be in the same zone as the network endpoint group.
-          custom_expand: 'templates/terraform/custom_expand/resource_from_self_link.go.tmpl'
+          custom_expand: 'templates/terraform/custom_expand/resource_from_self_link_nullable.go.tmpl'
```

```suggestion
          default_from_api: true
          custom_expand: 'templates/terraform/custom_expand/resource_from_self_link_nullable.go.tmpl'
```

This will make the field computed and prevent the null value from the config cause an empty diff when the API returns an empty string.